core/src/main/java/org/apache/accumulo/core/security/ColumnVisibility.java

```diff
@@ -125,6 +125,24 @@ public Node(int start, int end) {
       this.end = end;
     }
 
+    /**
+     * Creates a new node by performing a deep copy of an existing node object
+     *
+     * @param node Node object
+     * @since 2.1.4
+     */
+    public Node(Node node) {
```

Comment:
Since Node is removed in newer versions of accumulo this constructor could be switched to private to avoid standardizing its usage
```suggestion
    private Node(Node node) {
```